server/lobbyconnection.py

```diff
@@ -14,12 +14,13 @@
 import semver
 import server
 from server.db import FAFDatabase
-from sqlalchemy import and_, func, text
+from sqlalchemy import and_, func, select, text
 
 from . import config
 from .abc.base_game import GameConnectionState
 from .config import FAF_POLICY_SERVER_BASE_URL, TRACE, TWILIO_TTL
-from .db.models import ban, friends_and_foes
+from .db.models import ban, friends_and_foes, lobby_ban
+from .db.models import login as t_login
```

Comment:
Let's rename `t_login` to `table_login`? it's not something what you would like to read.

Reply:
I think it's clear from the context. When you see `select([t_login.c.login]...` you know what's going on. I would have left it as `login` but sadly that is already used to mean `username` in several places. I used `t_login` because that's what sqalcodegen does when it auto generates tables for you.